setup/docker/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+which mvn &> /dev/null || { echo >&2 "Maven package cant be found on path. Aborting."; exit 1; }
+which docker &> /dev/null  || { echo >&2 "Docker package cant be found on path. Aborting."; exit 1; }
+mvn package || { echo >&2 "Maven package has failed. Aborting."; exit 1; }
+
+export company="tananaev"
+export software="traccar"
+export version=$(head -n 10 ./pom.xml |grep version|cut -d ">" -f2|cut -d"<" -f1)
+
+tmp="./setup/docker/tmp"
+
+mkdir -p ${tmp}/data/database
```

Comment:
Well, then I suggest we split database changelog from database files, since while running external mapped folder we can upgrade container version (with news changelogs) while keeping old database. 
Its ok for you if I point in traccar.xml to this folder ?